packages/7_zip.rb

```diff
@@ -0,0 +1,55 @@
+require 'package'
+
+class PKG_7_zip < Package
```

Comment:
What would happen if this was changed to `class 7_zip < Package`?

Reply:
```
irb(main):009:1> class 7_zip < Package
irb(main):010:2*   def self.test
irb(main):011:1>   end
irb(main):012:0> end
/usr/local/lib64/ruby/3.1.0/irb/workspace.rb:119:in `eval': (irb):9: trailing `_' in number (SyntaxError)
class 7_zip < Package                                                           
       ^                                                                        
(irb):9: syntax error, unexpected local variable or method, expecting '.' or &. or :: or '['
class 7_zip < Package                                                           
        ^~~                                                                     
(irb):12: syntax error, unexpected `end', expecting end-of-input                
        from /usr/local/lib64/ruby/gems/3.1.0/gems/irb-1.4.1/exe/irb:11:in `<top (required)>'
        from /usr/local/bin/irb:25:in `load'                                    
        from /usr/local/bin/irb:25:in `<main>'     
```

Reply:
The example in #6748 is an wrong example:
> we are unable to name packages with a name that starts with digits (like class 7_zip < Package).